packages/pnpm/README.md

```diff
@@ -2,19 +2,19 @@
 
 ![](https://i.imgur.com/qlW1eEG.png)
 
-Fast, disk space efficient package manager:
+Fast, disk space-efficient package manager:
 
 * **Fast.** Up to 2x faster than the alternatives (see [benchmark](#benchmark)).
-* **Efficient.** Files inside `node_modules` are linked from a single content-addressable storage.
+* **Efficient.** Files inside `node_modules` are linked from single content-addressable storage.
 * **[Great for monorepos](https://pnpm.io/workspaces).**
 * **Strict.** A package can access only dependencies that are specified in its `package.json`.
 * **Deterministic.** Has a lockfile called `pnpm-lock.yaml`.
-* **Works everywhere.** Supports Windows, Linux, and macOS.
+* **Works everywhere.** It supports Windows, Linux, and macOS.
```

Comment:
This doesn't really improve grammar and makes the sentence structure inconsistent with the previous point.